src/index.html

```diff
@@ -16,7 +28,95 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="menu">
+      <a
+        class="logo"
+        href="#"
+      >
```

Comment:
Don't use spaces in `<a>` tag's `href` property. The `href` attribute for the logo link should start with `#` if it's meant to be a placeholder.